src/service/configs/fndlite.ts

```diff
@@ -0,0 +1,9 @@
+import { HelpConfig } from '../../help-config';
+
+export const config: HelpConfig = {
+  productId: 'fndlite',
+  hideHelpChat: true,
+  defaultHelpKey: 'tcs.html',
```

Comment:
Is this correct? I don't know what the fndlite svsid is or, but should it really point to the TCS help file?

Reply:
valid question. this is the same config from the [bb-help SPA](https://dev.azure.com/blackbaud/Products/_git/skyux-spa-bb-help?path=%2Fsrc%2Fhelp_configs%2Ffndlite.ts&version=GBmaster&_a=blame) so nothing's changing tho.

i _think_ at one point `fndlite` meant tcs, which is probably what was driving this. but i _think_ `fndlite` is _no longer_ a thing. i actually don't know.

Reply:
Makes sense. Sounds to me like we should just leave this as is. Thanks!